include/sys/zfs_stat.h

```diff
@@ -44,6 +44,8 @@ typedef struct zfs_stat {
 	uint64_t	zs_mode;
 	uint64_t	zs_links;
 	uint64_t	zs_ctime[2];
+	uint64_t	data_block_size;
```

Comment:
These should start with `zs_` like the rest of the fields in the struct; this is for code readability reasons.